### PR TITLE
Improved Fn::Cidr Documentation including examples of usage

### PR DESCRIPTION
**Pull request by NightKhaos**
*Description of changes:*

The actual name of this function is "Fn::Cidr" not "Fn::GetCidr". Additionally, we need better examples and the description of some of the parameters are quite confusing (particularly sizeMask).

This change request improves on the work done in #30 and #33 as a base and improves on it.

Detailed list of changes:

- Improved function definition
- Changed sizeMask to sizeBits to better reflect the purpose of the parameter
- Added detailed examples of usage including a common use case of setting up a VPC.
- Various wording improvements.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by alanthing** [comment]
Just wanted to add a "thank you" for this. I'm glad to see you removed mention of the 3rd argument being optional because it definitely is not (https://twitter.com/hellomichibye/status/975732356379267072).

**Comment by NightKhaos** [comment]
Fixed commit merged. Also restored "Supported Functions" as it had been dropped in my initial edit.

File: doc_source/intrinsic-function-reference-cidr.md [hunk 1]
```diff
@@ -1,95 +1,140 @@
...
+An array of CIDR address blocks\.
 
-### <a name="intrinsic-function-reference-cidr-example1"></a>
+## Supported Functions<a name="w3ab2c21c28c37c13"></a>
 
-#### YAML<a name="intrinsic-function-reference-cidr-example1.yaml"></a>
+You can use the following functions in a `Fn::Cidr` function:
+
++ `[`Fn::Select`](intrinsic-function-reference-select.md)` 
+
```
Comment (vgw-marc) [inline doc_source/intrinsic-function-reference-cidr.md]: These aren't rendering as links correctly e.g:
```
`[`Fn::Select`](intrinsic-function-reference-select.md)` 
```
 renders to `[`Fn::Select`](intrinsic-function-reference-select.md)` 

If you remove the wrapping '`' it should sort it:
```
[`Fn::Select`](intrinsic-function-reference-select.md)
```
renders to
[`Fn::Select`](intrinsic-function-reference-select.md)

Comment (NightKhaos) [reply doc_source/intrinsic-function-reference-cidr.md]: I'm not sure why this rendering is showing up incorrectly. The latest commit that we're trying to merge is correctly rendering and using the style you suggested in this comment. See:

https://raw.githubusercontent.com/NightKhaos/aws-cloudformation-user-guide/pr-33/doc_source/intrinsic-function-reference-cidr.md

**Comment by jdub** [comment]
This change greatly improves the current documentation. `Fn::Cidr` makes much more sense having found this PR. 😄

**Comment by NightKhaos** [comment]
This looks like it has been merged, but with some minor fixes. Can this PR be closed then?

**Pull request closed without merging**